### ISSUE TITLE
macOS accessibility attributes are missing

### ISSUE THREAD

**Issue by ivmirx** [adobe/brackets#12868]
### Description

Brackets is not reporting its window information to macOS accessibility.

### Steps to Reproduce

1. run Accessibility Inspector from Xcode
2. focus Brackets window with an opened file

**Expected behavior:** all the attributes like Title, Document should contain values

**Actual behavior:** `<nil>` or `<empty string>` in these attributes

### Versions

macOS 10.11.6
Brackets Release 1.7 versión 1.7.0-16898 (release b0a363b71) 

### Accessibility Inspector screenshot

![untitled-1](https://cloud.githubusercontent.com/assets/10554114/19893621/90ab28ea-a028-11e6-8369-1aebe4f9ae4b.png)

**Comment by mmcken3** [comment]
I can confirm that I have replicated and seen the same issue on Mac OS 10.10.5 and Brackets 1.7 version 1.7.0-16898.

**Comment by ficristo** [comment]
Out of curiosity, could you give a try to https://github.com/zaggino/brackets-electron/ to see if it fixes this?

**Comment by ivmirx** [comment]
@ficristo it shows Title now but it's "Brackets-Electron" instead of full title "index.html (Getting Started) – Brackets-Electron".

The "Document" parameter is still `nil`, it should match the currently opened file path, something like `file:///Users/MyName/Documents/example.js`.
